update contribution docs (#1496)
shopware/docs: resources/guidelines/code/contribution.md

* A pull request to the Shopware core always has to be made to the [main shopware](https://github.com/shopware/shopware) repository.
* Fill out the [pull request info template](https://github.com/shopware/shopware/blob/trunk/.github/PULL_REQUEST_TEMPLATE.md) as detailed as possible.
* Create a changelog file with documentation of your changes. Refer to [Changelog](https://github.com/shopware/shopware/blob/master/adr/2020-08-03-implement-new-changelog.md) section for more detailed information about writing changelog.

> * Check if your pull request addresses the correct Shopware branch: 6.5.x for upcomming 6.5 version and trunk for next upcomming 6.6 version.

Reviewer: ```suggestion
* Check if your pull request addresses the correct Shopware branch: 6.5.x for upcoming 6.5 version and trunk for next upcoming 6.6 version.
```

|        ![GitHub label quickpick](../../../assets/github-label-quickpick.png)         | The changes you provide seem to be easy to test and implement. Our developers decided to integrate this quickly into our software. There will probably be no ticket for this change to follow, but you will be informed by the accepted label on this pull request that your change was finally merged into the product. |
|         ![GitHub label accepted](../../../assets/github-label-accepted.png)          | Your changes are finally accepted. The pull request passed our internal workflow. Your changes will be released with one of the next releases. |
|    ![GitHub label missing tests](../../../assets/github-label-missing-tests.png)     | Your pull request lacks the necessary tests for your changes. E.g. [Jest](../../../guides/plugins/plugins/testing/jest-admin) or [Cypress](../../../guides/plugins/plugins/testing/end-to-end-testing) tests for frontend changes or [PHPUnit](../../../guides/plugins/plugins/testing/php-unit) tests for backend changes. |

## What happens after a pull request has been created

> Everyday weekdays, we assign the pull request to a Area (team), which is responsible for the specific part of the Shopware software. The Area will then review further your pull request and decide what to do next.

Reviewer: ```suggestion
Everyday weekdays, we assign the pull request to an Area (team) which is responsible for the specific part of the Shopware software. The Area will then review your pull request and decide what to do next.
```

The Area can either accept your pull request, decline it, or ask you to update it with more information or changes.

> The next step is that the pull request get's an "Scheduled" label.

Reviewer: ```suggestion
The next step is that the pull request gets a "Scheduled" label.
```
